thesis-de/content-de.tex

```diff
@@ -1708,9 +1714,9 @@ \subsubsection{Durchführung Evaluierung Frontend}
 \label{paragraph:gridster-d3}
 Gidster ist eine von Anmol Koul entwickelte Softwarelösung, welche auf
 OpenSource Basis auf \gls{GitHub} liegt. Während der Evaluierung von dieser
-Software, ist aufgefallen, dass es sich hierbei nicht direkt um eine
-Visualisieurngssoftware handelt, sondern lediglich eine Funktion innerhalb
-einer Weboberfläche, nämlich das Drag \& Drop, von einzelnen Graphen.
+Software ist aufgefallen, dass es sich hierbei nicht direkt um eine
+Visualisierungssoftware handelt, sondern lediglich eine Funktion, nämlich das
```

Comment:
, dem Drag \& Drop -> und dann im Glossar kurz Drag & Drop erläutern -> dann kannst du das nämlich weg lassen

(Ist nur im Vorschlag der mir gerade aufgefallen ist, macht den Satz noch smarter -> ist mir beim ersten Comment nicht aufgefallen sorry)

(Da kommt mir nur immer der Satz aus der Schule in Kopf :D "Wer nämlich sagt ist dämlich")